packages/contracts-bedrock/contracts/universal/StandardBridge.sol

```diff
@@ -344,18 +362,34 @@ abstract contract StandardBridge {
         bytes calldata _extraData
     ) public onlyOtherBridge returns (bool) {
         try this.completeOutboundTransfer(_localToken, _remoteToken, _to, _amount) {
-            emit ERC20BridgeFinalized(_localToken, _remoteToken, _from, _to, _amount, _extraData);
-            return true;
+            if (_from == address(otherBridge)) {
+                // The _from address is the address of the remote bridge if a transfer fails to be
+                // finalized on the remote chain.
+                // slither-disable-next-line reentrancy-events
+                emit ERC20BridgeRefunded(_localToken, _remoteToken, _to, _amount, _extraData);
+                return true;
+            } else {
+                emit ERC20BridgeFinalized(
+                    _localToken,
+                    _remoteToken,
+                    _from,
+                    _to,
+                    _amount,
+                    _extraData
+                );
+                return true;
```

Comment:
it would be better to move these duplicate return statements to the end of the if/else block

Reply:
Yes plz